src/ClassyResource/_refreshAppToken.js

```diff
@@ -4,18 +4,43 @@
  * @return {[type]} [description]
  */
 export default function _refreshAppToken() {
-  const promise = new Promise((resolve, reject) => {
-    if (!this._classy.appToken) {
-      this._classy.getAppToken().then((response) => {
-        this._classy.setAppToken(response);
-        resolve(response);
-      }, (error) => {
-        reject(error);
+  try {
```

Comment:
this one seems redundant because the promise will handle any errors that occur inside it